plugins/orchestrator/src/api/MockOrchestratorClient.ts

```diff
@@ -29,6 +30,7 @@ export interface MockOrchestratorApiData {
     OrchestratorApi['getWorkflowDataInputSchema']
   >;
   listWorkflowsResponse: ReturnType<OrchestratorApi['listWorkflows']>;
+  getWorkflowOverview: ReturnType<OrchestratorApi['getWorkflowOverview']>;
```

Comment:
```suggestion
  getWorkflowOverviewResponse: ReturnType<OrchestratorApi['getWorkflowOverview']>;
```

Reply:
why this rename?

Reply:
For consistency. Please, pay attention to the names of the other members in this interface